kc-cloudtrust-testsuite/kc-cloudtrust-test-tools/src/main/java/io/cloudtrust/keycloak/test/matchers/EventMatchersLegacy.java

```diff
@@ -0,0 +1,146 @@
+package io.cloudtrust.keycloak.test.matchers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.lang3.StringUtils;
+import org.hamcrest.BaseMatcher;
+import org.keycloak.events.EventType;
+import org.keycloak.representations.idm.EventRepresentation;
+
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+// Use keycloak test framework with EventMatchers. EventMatchersLegacy will be removed when KeycloakDeploy won't be used anymore
```

Comment:
Should we do a jira to not forget to remove legacy classes ?

Reply:
Should be part of the changes to do when we will migrate to Keycloak test framework (current tools should be adapted and old tools should be removed)